client/my-sites/post-relative-time-status/style.scss

```diff
@@ -22,7 +22,10 @@
 	.is-trash {
 		color: var(--color-error);
 	}
-	.is-private {
+	.is-private,
```

Comment:
Can we add more specificity in the name?  Something like:

`is-newsletter-private`
`is-newsletter-subscribers`
`is-newsletter-paid-subscribers`

Reply:
Fixed in [0d1c298](https://github.com/Automattic/wp-calypso/pull/76382/commits/0d1c298442251e9a1738ee0359a026263e73f2e1)